sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesResult.cs

```diff
@@ -16,11 +17,11 @@ public partial class AnalyzeHealthcareEntitiesResult : TextAnalyticsResult
 
         internal AnalyzeHealthcareEntitiesResult(string id, TextDocumentStatistics statistics,
             IReadOnlyList<HealthcareEntity> healthcareEntities,
-            IReadOnlyList<TextAnalyticsWarning> warnings)
+            IList<TextAnalyticsWarning> warnings)
             : base(id, statistics)
         {
             _entities = healthcareEntities;
```

Comment:
this should also be initialized, like you did with the warnings. We always need to do this for any reference type